wiki/People/Users_with_unique_titles/en.md

```diff
@@ -85,10 +85,16 @@ Most official mapping contests offer the **Elite Mapper** title as a first-place
 
 Since 2020, [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators)' and [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) members' activity in supporting the ranking system has been rewarded with the **Elite Nominator** title. This title is given at the beginning of each year for contributions made during the last, and is removed upon the user leaving the team.
 
+- ![][flag_CA] [Agatsu](https://osu.ppy.sh/users/5579871)
+- ![][flag_JP] [BaAR_Vendel](https://osu.ppy.sh/users/8679346)
 - ![][flag_TW] [bossandy](https://osu.ppy.sh/users/360437)
```

Comment:
no longer has the title

Reply:
Fixed in ad53d082337b564303d7026d5a652d211566538a